.eslintrc.js

```diff
@@ -0,0 +1,103 @@
+module.exports = {
+	extends: [
+		'wikimedia',
+		'wikimedia/node',
+		'wikimedia/language/rules-es2017',
+	],
+	plugins: [
+		'@typescript-eslint',
+		'filenames',
+	],
+	parser: '@typescript-eslint/parser',
+	parserOptions: {
+		parser: '@typescript-eslint/parser',
+		sourceType: 'module',
+		ecmaFeatures: {
+			impliedStrict: true,
+		},
+	},
+	root: true,
+	rules: {
+		// We need a shared TS eslint config; copy from data-bridge
```

Comment:
This is now in termbox, bridge, tainted references, and here in one version or another.